pydantic/env_settings.py

```diff
@@ -79,15 +79,15 @@ def _build_values(
             return {}
 
     class Config(BaseConfig):
-        env_prefix = ''
-        env_file = None
-        env_file_encoding = None
-        env_nested_delimiter = None
-        secrets_dir = None
-        validate_all = True
-        extra = Extra.forbid
-        arbitrary_types_allowed = True
-        case_sensitive = False
+        env_prefix: str = ''
```

Comment:
can't we use `ClassVar` here?

Reply:
maybe, but is there any advantage?

Reply:
probably nothing :)

Reply:
I think given this will be either the final beta, or included directly in the release, I want to keep it as simple as possible.